algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/model.py

```diff
@@ -378,8 +368,10 @@ def forward(self,
       if not why_not_fast_path:
```

Comment:
can we simplify this logic related to why_not_fast_path ?

Reply:
That's the code written by pytorch developers and I have left it as is because it will be easier to diff the changes!